101-cosmosdb-create-arm-mongo/metadata.json

```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Mongo API", 
+   "description": "This template creates a Cosmos Database Account using the Mongo API",
```

Comment:
Please change to "This template creates an Azure Cosmos DB MongoDB API account."